types/react-close-on-escape/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for react-close-on-escape 3.0
+// Project: https://github.com/conorhastings/react-close-on-escape#readme
+// Definitions by: Robert Baruck <https://github.com/JamesAlias>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+import * as React from "react";
+
+export interface CloseOnEscapeProps {
+    /**
+     * Callback triggered by pressing ESC
+     */
+    onEscape(): void;
+}
+
+export class CloseOnEscape extends React.Component<CloseOnEscapeProps> {}
```

Comment:
As per [this](https://github.com/conorhastings/react-close-on-escape#use) it needs to be exported as default instead.